docs/api.md

```diff
@@ -437,6 +437,68 @@ abs_l1
 abs_l2
 ```
 
+#### `TENSORBOARD_CONFIG_FILE_PATH`:
```

Comment:
Wait these should be moved to env_var.md file. I guess the merge conflict wasn't resolved correctly